src/memote/suite/tests/test_consistency.py

```diff
@@ -51,32 +51,82 @@ def test_stoichiometric_consistency(model):
     publication. In addition, the list of min unconservable sets is computed
     using the algorithm described in section 3.3.
 
+    """
+    ann = test_stoichiometric_consistency.annotation
+    is_consistent = consistency.check_stoichiometric_consistency(model)
+    ann["data"] = is_consistent
+    ann["metric"] = 1.0 - float(is_consistent)
+    ann["message"] = wrapper.fill(
+        """This model is stoichiomerically {}""".format(
+            "consistent" if is_consistent else "inconsistent"
+        )
+    )
+    assert is_consistent, ann["message"]
+
+
+@annotate(title="Uncoserved Metabolites", format_type="percent")
```

Comment:
As mentioned, I'd move this into the unscored part of the report. Thus a good format type would be `"count"`.